gt/main.c

```diff
@@ -2211,3 +2211,27 @@ l_update_gt_lua_states_incrementally(lua_State *l)
 
 	return 0;
 }
+
+uint16_t
+gt_cpu_to_be_16(uint16_t x)
```

Comment:
Add the following comment to this function:
`/* This function is only meant to be used in Lua policies. If you need it in Gatekeeper's C code, use rte_cpu_to_be_16() */`
Break the comment as needed.

Add analogous comments to the other three functions being added.